src/widgetastic_patternfly4/modal.py

```diff
@@ -22,6 +22,8 @@ class Modal(Widget):
     )
     TITLE = ".//h1[contains(@class, 'pf-c-title')]"
     CLOSE = ".//button[@aria-label='Close']"
+    NOTIFICATIONS = ".//div[contains(@class, 'pf-c-alert') and contains(@class, 'notification')]"
+    NOTIFICATION_ACTION = ".//div[contains(@class, 'pf-c-alert__action')]"
```

Comment:
Are these alerts children of Modal `ROOT`? Moreover, we have Alert widget implemented already in https://github.com/RedHatQE/widgetastic.patternfly4/blob/master/src/widgetastic_patternfly4/alert.py. Perhaps we may redesign it and make it as `ParametrizedView`.

Reply:
No they are not children of the Modal `ROOT`, but the problem is that they will sometimes block the `close` button on the modal. I will take a look at redesigning it as a `ParemetrizedView`